src/api/modules/orgs/helpers/org-groups-requests.ts

```diff
@@ -504,3 +560,57 @@ export const verifyOrgGroupRequest = async ({
 
   return data
 }
+
+export const rejectOrgGroupRequest = async ({
+  orgId,
+  groupId,
+  reqId,
+}: {
+  orgId: string
+  groupId: string
+  reqId: string
+}) => {
+  const { data } = await api.post<OrgGroupRequest>(
+    `/v1/orgs/${orgId}/groups/${groupId}/requests/${reqId}`,
+    {
+      body: {
+        data: {
+          type: 'requests-verify',
+          attributes: {
+            approved: true,
+          },
+        },
+      },
+    },
+  )
+
+  return data
+}
+
+export const parseRequestCredentialSchemas = async (
```

Comment:
IMO parse function should be sync

here you don't only parse the schemas but fetch them as well